src/main/java/org/zalando/nakadi/domain/EventTypeOptions.java

```diff
@@ -1,10 +1,7 @@
 package org.zalando.nakadi.domain;
 
-import javax.annotation.Nullable;
-
 public class EventTypeOptions {
 
-    @Nullable
     private Long retentionTime;
```

Comment:
We use default values from application.yml to be able to change this values without recompiling (using env vars). Maybe I did not get your point.